sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -2211,7 +2211,7 @@ class ColumnExpressionSuite extends QueryTest with SharedSparkSession {
     intercept[AnalysisException] {
       sql("SELECT named_struct('a', named_struct('b', 1), 'a', named_struct('c', 2)) struct_col")
         .select($"struct_col".dropFields("a.c"))
-    }.getMessage should include("Ambiguous reference to fields")
+    }.getMessage should include("Ambiguous reference to the field `a`")
```

Comment:
Don't check error message, use `checkError` instead of it.

Reply:
Done.